apps/backend/src/models/proxy_config.py

```
@@ -0,0 +1,264 @@
+    """
+    try:
+        from apps.backend.src.models.device import Device
+        if not hasattr(Device, 'proxy_configs'):
+            Device.proxy_configs = relationship("ProxyConfig", back_populates="device", cascade="all, delete-orphan")
+    except ImportError:
+        # Device model not available yet, will be added later
+        pass
+
+
+# Call the relationship setup
+add_proxy_config_relationship()
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)